flink-formats/flink-avro-confluent-registry/src/test/java/org/apache/flink/formats/avro/registry/confluent/debezium/DebeziumAvroFormatFactoryTest.java

```diff
@@ -0,0 +1,138 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.formats.avro.registry.confluent.debezium;
+
+import org.apache.flink.api.common.serialization.DeserializationSchema;
+import org.apache.flink.api.common.serialization.SerializationSchema;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.formats.avro.registry.confluent.RegistryAvroOptions;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.table.catalog.CatalogTableImpl;
+import org.apache.flink.table.catalog.ObjectIdentifier;
+import org.apache.flink.table.connector.sink.DynamicTableSink;
+import org.apache.flink.table.connector.source.DynamicTableSource;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.factories.FactoryUtil;
+import org.apache.flink.table.factories.TestDynamicTableFactory;
+import org.apache.flink.table.runtime.connector.sink.SinkRuntimeProviderContext;
+import org.apache.flink.table.runtime.connector.source.ScanRuntimeProviderContext;
+import org.apache.flink.table.runtime.typeutils.InternalTypeInfo;
+import org.apache.flink.table.types.logical.RowType;
+import org.apache.flink.util.TestLogger;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.apache.flink.core.testutils.FlinkMatchers.containsCause;
+
+/**
+ * Tests for {@link DebeziumAvroFormatFactory}.
+ */
+public class DebeziumAvroFormatFactoryTest extends TestLogger {
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	private static final TableSchema SCHEMA = TableSchema.builder()
+		.field("a", DataTypes.STRING())
+		.field("b", DataTypes.INT())
+		.field("c", DataTypes.BOOLEAN())
+		.build();
+
+	private static final RowType ROW_TYPE = (RowType) SCHEMA.toRowDataType().getLogicalType();
+
+	private static final String SUBJECT = "test-debezium-avro";
+	private static final String REGISTRY_URL = "http://localhost:8081";
+
+	@Test
+	public void testSeDeSchema() {
+		final Map<String, String> options = getAllOptions();
+		String schemaRegistryUrl = options.get(DebeziumAvroFormatFactory.IDENTIFIER + "." + RegistryAvroOptions.SCHEMA_REGISTRY_URL.key());
+		String schemaRegistrySubject = options.get(DebeziumAvroFormatFactory.IDENTIFIER + "." + RegistryAvroOptions.SCHEMA_REGISTRY_SUBJECT.key());
+
+		final DebeziumAvroDeserializationSchema expectedDeser = new DebeziumAvroDeserializationSchema(
+			ROW_TYPE,
+			InternalTypeInfo.of(ROW_TYPE),
+			schemaRegistryUrl);
```

Comment:
Use `REGISTRY_URL` instead of extract the value from options. This is used for asserting which should be a constant value.